### ISSUE TITLE
[MCP] Email copy/design enhancements for V2 implementation

### ISSUE THREAD

**Issue by Tom-Davis** [department-of-veterans-affairs/va.gov-team#61324]
## Issue Description
The MCP notification email copy lacks some key information regarding copays. Needs to be edited to have a more clear CTA and be more personal and relevant to a veteran. 

This is a continuation of ticket #59097

---
## Tasks
- [x] Update copy and subject line to be more personal and actionable.
- [x] Update stakeholders with the status of implementation ([Slack thread is here](https://dsva.slack.com/archives/CPE4AJ6Q0/p1685053653718719))

## Acceptance Criteria
- [x] The updated copy is approved

**Comment by HeatherRienks** [comment]
Per @denisecoveyduc - need to prioritize this. Veterans are confused and need link to log in. Will discuss in planning. Need to update soon!

**Comment by mgleo** [comment]
Issues
- Users cannot opt out of emails
- Some users noted that the emails are alarming and confusing
- Users may not be receiving up to date emails and information on the debt portal may not be up to date.
    - Can a date be added to the email?
- Information in email should be more specific to prevent confusion:
    - Can we specify this email is for a copay?
    - Link directly to the manage-va-debt/summary/copay-balances

**Comment by mgleo** [comment]
From @denisecoveyduc regarding opt out:

> Sounds like Veterans do have a way to set email notification preferences in their profile. It isn't there for debt notifications yet but we can request it.  It isn't work our team would do so I am getting a sense of timing.
I'd recommend we don't consider this for the initial copy updates but for the next round.

**Comment by mgleo** [comment]
@denisecoveyduc and @josephrlee, Pulling from Joe's original suggestions and adding the direct link to Copays and if possible it would be nice to also let them know the point in time from when this information is pulled from. If they paid their copay on 01/10, it would be helpful to know the notification is from 01/02 and not a new copay.

 Current subject line: Important information from the Veterans Health Administration
Proposed subject line: **You have a new medical copay from the Veterans Health Administration**
Then the body would read like so:
Dear [First Name],
You have a new medical copay from the Veterans Health Administration. Please sign in to https://va.gov/manage-va-debt/summary/copay-balances to view the details.
[View your copay details](Big button linked to https://va.gov/manage-va-debt/summary/copay-balances)
You will also receive a letter from the VHA with details of this information.
This information is up to date as of January 2, 2024.
Thank you,
Veterans Affairs

**Comment by josephrlee** [comment]
The date referenced in the above email copy would be pulled from the **Statement date of the copay bill**

**Comment by HeatherRienks** [comment]
Waiting on approval!

**Comment by HeatherRienks** [comment]
Updated copy due to be merged into master today (#74795). Will create new ticket for button work.

**Comment by denisecoveyduc** [comment]
@HeatherRienks Who can clarify what the "button work" is? I think I know just want to make sure I am tracking. Maybe @mgleo or @josephrlee ?

**Comment by mgleo** [comment]
@josephrlee Shared this slack thread: https://dsva.slack.com/archives/C010R6AUPHT/p1693331855604319

Looks like their solution was not a button but a link with with lines below and above the link to draw attention to it as the main call to action. 
We'd need to speak with the VANotify team more to understand the restrictions if we wanted to continue with a traditional button.

**Comment by HeatherRienks** [comment]
We will add to 2/26 Design Sync.

**Comment by HeatherRienks** [comment]
Closing this as V2 copy updates have been made.
